src/celengine/modelgeometry.cpp

```diff
@@ -228,6 +228,7 @@ ModelGeometry::loadTextures()
 }
 
 
+/*
```

Comment:
please remove this instead of commenting out

Reply:
Oops, had that on my to-do list but missed it. Done.